Snippets/CloudServicesHost/CloudServicesHost_8/AzureConfigurationSourceReplacement.cs

```diff
@@ -0,0 +1,242 @@
+﻿using NServiceBus;
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection;
+using System.Configuration;
+using System.Web.Configuration;
+using System.Web.Hosting;
+
+class AzureConfigurationSourceReplacement
+{
+    AzureConfigurationSourceReplacement(EndpointConfiguration endpointConfiguration)
```

Comment:
`endpointConfiguration` is not used, remove?

Reply:
Hmmm, I think a snippet could be something like 

```c#
string GetConfigurationSection(string sectionName, string attributeName)
{
  // awesome code you provided
  return value;
}
```

I can guarantee people just going to copy/paste and use it as-is. So why to force them to create their own methods and go an extra step just to re-use the code? 🙂